README.md

```diff
@@ -57,7 +57,8 @@ Now you can start the server:
         --bind :8000 \
         --resource-manager-url http://resourcemanager:8088 \
         --history-server-url http://resourcemanager:19888 \
-        --namenode-address namenode:9000
+        --namenode-address namenode:9000 \
+        --statsd-address localhost:8200
```

Comment:
It's worth noting in your docs that this is specifically [DogStatsD](https://docs.datadoghq.com/guides/dogstatsd/) and not generic statsd.  Also note that outside of Stripe most people emit DogStatsD to port `8125`. That might help folks!